mdai/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .client import Client
 from .utils import common_utils
 from .utils import transforms
+from .utils import SR_SEG_utils
```

Comment:
Module names should have short, all-lowercase names (see [pep8](https://peps.python.org/pep-0008/#package-and-module-names)).